### PR TITLE
Add example to crate docs

### PR DESCRIPTION
**Pull request by pickfire**
Example stolen from README

### CODE REVIEW

File: src/lib.rs [hunk 1]
```diff
@@ -1,4 +1,21 @@
 //! Cross-platform interface to the `errno` variable.
+//!
+//! # Example
```
[the remark this answers is not in the document]
Comment (pickfire) [reply src/lib.rs:3]: Okay, thanks.

File: src/lib.rs [hunk 2]
```diff
@@ -1,4 +1,21 @@
...
+//! ```edition2018
+//! use errno::{self, Errno, set_errno};
```
[the remark this answers is not in the document]
Comment (pickfire) [reply src/lib.rs:5]: What is was before?

File: src/lib.rs [hunk 3]
```diff
@@ -1,4 +1,21 @@
 //! Cross-platform interface to the `errno` variable.
+//!
+//! # Example
+//! ```edition2018
```
[the remark this answers is not in the document]
Comment (pickfire) [reply src/lib.rs]: I use \``` since it is the same as ```rust.

File: src/lib.rs (hunk 2 above)
[the remark this answers is not in the document]
Comment (lambda-fairy) [reply src/lib.rs:5]: In the readme, it was

```rust
use errno::{Errno, errno, set_errno};
```

Did you change it?

[the remark this answers is not in the document]
Comment (pickfire) [reply src/lib.rs:5]: Ah, okay.

**Pull request merged**